cli/defaults/siteConfig.json

```diff
@@ -42,11 +42,43 @@
   },
   "serving": {
     "markdownSettings": {
-      "skipHtml": true,
-      "privilegedDisallowedTypesDescriptions": ["Image"],
-      "privilegedDisallowedTypesMain": [],
-      "publicDisallowedTypesDescriptions": ["Image"],
-      "publicDisallowedTypesMain": []
+      "skipHtmlMain": false,
+      "escapeHtmlMain": false,
```

Comment:
I took a closer look, set this to `true` and make sure it's documented as a dangerous setting.  
I don't want users deploying making mistakes, this needs to be explicitly `true` on all our instances that allow public content.